lte/cloud/go/services/subscriberdb/streamer/providers.go

```diff
@@ -35,13 +36,17 @@ func (p *SubscribersProvider) GetStreamName() string {
 }
 
 func (p *SubscribersProvider) GetUpdates(gatewayId string, extraArgs *any.Any) ([]*protos.DataUpdate, error) {
-	gateway, err := configurator.LoadEntityForPhysicalID(gatewayId, configurator.EntityLoadCriteria{LoadAssocsFromThis: true})
+	magmadGateway, err := configurator.LoadEntityForPhysicalID(gatewayId, configurator.EntityLoadCriteria{LoadAssocsFromThis: true})
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrapf(err, "load magmad gateway for physical ID %s", gatewayId)
+	}
+	gateway, err := configurator.LoadEntity(magmadGateway.NetworkID, lte.CellularGatewayEntityType, magmadGateway.Key, configurator.EntityLoadCriteria{LoadAssocsFromThis: true})
```

Comment:
This line is the fix -- make `gateway` a cellular_gateway, rather than magmad_gateway